src/main/java/core/basesyntax/interfaces/DrawFigure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.interfaces;
+
+public interface DrawFigure {
+
+    void toDrawFigure();
```

Comment:
```suggestion
public interface Drawble {
    void drawFigure();
```
Don’t use verbs for class names. Remove redundant empty lines. Avoid  using  articles  in variable (method) names

Reply:
no